internal/server/audit/audit.go

```diff
@@ -151,7 +151,7 @@ func decodeToEvent(kvs []attribute.KeyValue) (*Event, error) {
 		case eventTimestampKey:
 			e.Timestamp = kv.Value.AsString()
 		case eventMetadataActorKey:
-			var actor map[string]string
+			var actor *Actor
```

Comment:
```suggestion
			var actor Actor
```
i think this will panic as it would point to nil no?